official-antistasi-community/A3-Antistasi
# Better GC Tracking

1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:

Created new way to keep track of GC, notify about time since Gc and trigger auto GC.
It now keeps track of when an manual GC happens and restarts the loop also it is done so, it is easy to change. 	

Changed the `disabled` value from 99999 to 0, because if 0 is a viable value it would probably break the sleeps or block the server to hell.

Added `time until Auto-GC` to the reminder and also added an `final warning` that happens 2 minutes before an auto-GC.
    
Discussion:

To give the player the `time until Auto-GC` I had to change the Stringtable text, because I do not know the translations of this new text, I just added the second parameter to it. Is there a special way how we should handle those kind of situations?


### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
- start a game
- set the auto-GC threshold to a value that is not `disabled`
- check if the reminder, the final-warning and the auto-GC happens
- do a manual GC and check if the auto-GC happens after the set value in relation to this manual GC.
********************************************************
Notes:
